rescheduler/rescheduler.go

```diff
@@ -373,11 +382,17 @@ func groupPods(client *kube_client.Client, node *kube_api.Node) ([]*kube_api.Pod
 
 		if ca_simulator.IsMirrorPod(pod) || creatorRef == "DaemonSet" || isCriticalPod(pod) {
 			requiredPods = append(requiredPods, pod)
+		} else if bucketList, ok := podBucketMap[pod.GetNamespace()]; ok {
+			bucketList = append(bucketList, pod)
 		} else {
 			otherPods = append(otherPods, pod)
 		}
 	}
 
+	for _, bucketName := range *podBucketList {
+		otherPods = append(otherPods, podBucketMap[bucketName]...)
+	}
+
```

Comment:
The project already uses glog, so it supports --v=N.
So you could add `glog.V(2).Infof("Required pods: %v, other pods: %v", requiredPods, otherPods)` here, allowing people to run the rescheduler with --v=2 to get more details on what's going on.